app/views/transaction.py

```diff
@@ -0,0 +1,766 @@
+            return response.success_response()
+        except FundTransferObjectNotFound as err:
+            logger.exception(err.message)
+            response = ResponseGenerator(data={},
+                                         message=err.message,
+                                         success=False,
+                                         status=HTTPStatus.NOT_FOUND)
+            return response.error_response()
+
+    def put(self, fund_transfer_id):
+        """
+             This is PUT API
+             Call this api passing a fund transfer id
+             parameters:
+                 source: String
+                 destination: String
+             responses:
+                 404:
+                     description: Fund transfer with this id does not exist
+                 200:
+                     description: Fund transfer with this id updated successfully
+                     schema:
+                         FundTransferSchema
+         """
+        try:
+            # retrieve body data from input JSON
+            data = request.get_json()
+            errors = fund_transfer_schema.validate(data, partial=True)
+            if errors:
+                logger.error("Missing or sending incorrect data to create an activity {}".format(errors))
+                response = ResponseGenerator(data={},
+                                             message=errors,
+                                             success=False,
+                                             status=HTTPStatus.BAD_REQUEST)
+                return response.error_response()
+
+            fund_transfer_data = FundTransfer.query.filter(FundTransfer.id == fund_transfer_id).first()
+            if not fund_transfer_data:
+                raise FundTransferObjectNotFound("Fund transfer with this id does not exist")
+
+            fund_transfer_data.from_account = data.get('from_account', fund_transfer_data.from_account)
+            fund_transfer_data.to_account = data.get('to_account', fund_transfer_data.to_account)
+
+            db.session.commit()
+            result = fund_transfer_schema.dump(fund_transfer_data)
+
+            logger.info("Response for put with id request for fund transfer {}".format(result))
+            response = ResponseGenerator(data=result,
+                                         message="Fund transfer with this id updated successfully",
+                                         success=True,
+                                         status=HTTPStatus.OK)
+            return response.success_response()
+        except FundTransferObjectNotFound as err:
+            logger.exception(err.message)
+            response = ResponseGenerator(data={},
+                                         message=err.message,
+                                         success=False,
+                                         status=HTTPStatus.NOT_FOUND)
+            return response.error_response()
+
+    def delete(self, fund_transfer_id):
```

Comment:
We do not need fund transfer delete API I think.

Reply:
Made the changes as suggested.